new-lamassu-admin/src/pages/Dashboard/SystemPerformance/Graphs/PercentageChart.js

```diff
@@ -0,0 +1,101 @@
+import { makeStyles } from '@material-ui/core'
+import React from 'react'
+
+import { ReactComponent as CashIn } from 'src/styling/icons/direction/cash-in.svg'
+import { ReactComponent as CashOut } from 'src/styling/icons/direction/cash-out.svg'
+import {
+  zircon,
+  fontSize3,
+  fontSecondary,
+  fontColor
+} from 'src/styling/variables'
+
+const styles = {
+  wrapper: {
+    display: 'flex',
+    height: 130,
+    marginTop: -8
+  },
+  percentageBox: {
+    backgroundColor: zircon,
+    height: 130,
+    borderRadius: 4,
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    whiteSpace: 'pre'
+  },
+  label: {
+    fontSize: fontSize3,
+    fontFamily: fontSecondary,
+    fontWeight: 700,
+    color: fontColor
+  }
+}
+
+const useStyles = makeStyles(styles)
+
+const PercentageChart = ({ data }) => {
+  const classes = useStyles()
+  const value = data.cashIn && data.cashOut !== 0 ? data.cashIn : 50
+
+  const buildPercentageView = (value, direction) => {
+    switch (direction) {
+      case 'cashIn':
+        if (value > 25) {
+          return (
+            <>
+              <CashIn />
+              <span className={classes.label}>{` ${value}%`}</span>
+            </>
+          )
+        }
+        if (value >= 10) {
+          return (
+            <>
+              <CashIn />
+            </>
+          )
+        }
+        return null
+      case 'cashOut':
+        if (value > 25) {
+          return (
+            <>
+              <CashOut />
+              <span className={classes.label}>{` ${value}%`}</span>
+            </>
+          )
+        }
+        if (value >= 10) {
+          return (
+            <>
+              <CashOut />
+            </>
+          )
+        }
+        return null
+      default:
+        return null
+    }
+  }
```

Comment:
Maybe something like this would be better:

`    const Operation = direction == 'cashIn' ? CashIn : CashOut;
    
    return (value >= 10 &&
      <>
        <Operation />
        {value > 25 && <span className={classes.label}>{` ${value}%`}</span>}
      </>
    )`